gapis/api/vulkan/vulkan.api

```diff
@@ -256,6 +260,18 @@ sub ref!ExtensionSet supportedDeviceExtensions() {
   supported.ExtensionNames["VK_KHR_descriptor_update_template"] = true
   supported.ExtensionNames["VK_KHR_device_group"] = true
   supported.ExtensionNames["VK_GOOGLE_display_timing"] = true
+  supported.ExtensionNames["VK_EXT_depth_range_unrestricted"] = true
+  supported.ExtensionNames["VK_EXT_pci_bus_info"] = true
+  supported.ExtensionNames["VK_EXT_shader_stencil_export"] = true
+  supported.ExtensionNames["VK_EXT_shader_subgroup_ballot"] = true
+  supported.ExtensionNames["VK_EXT_shader_subgroup_vote"] = true
+  supported.ExtensionNames["VK_EXT_shader_viewport_index_layer"] = true
+  supported.ExtensionNames["SPV_KHR_storage_buffer_storage_class"] = true
```

Comment:
This is wrong. The Vulkan extension is `VK_KHR_storage_buffer_storage_class`.

Reply:
(SPV_KHR_storage_buffer_storage_class is the SPIRV part)

Reply:
thanks!